package.json

```diff
@@ -1,12 +1,9 @@
 {
   "name": "@snyk/sweater-comb",
-  "packageManager": "yarn@3.0.2",
```

Comment:
I'm removing this, I'm not sure what was happening before, but the `yarn.lock` file was yarn v1 (so technically this project was always on yarn v1), and for some reason only now my yarn runner wanted to upgrade to v3.

To keep this PR focused, I'm removing this so that the lock file stays on v1 and that other people running this won't upgrade to yarn v3 unless it's intentional